Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Bindings-Forms.swift

```diff
@@ -16,7 +16,7 @@ private let readMe = """
 // MARK: - Feature domain
 
 struct BindingForm: ReducerProtocol {
-  struct State: BindableStateProtocol, Equatable {
+  struct State: BindableViewState, Equatable {
```

Comment:
This can be dropped since no view state is used right?

```suggestion
  struct State: Equatable {
```